tests/cmd/tuto/in/main.rs

```diff
@@ -2,6 +2,7 @@ mod protos {
     tonic::include_proto!("buffrs");
 }
 
+#[allow(dead_code)]
```

Comment:
This code is only defined in the code that's compiled as part of the tutorial test. The warning only pops up as part of `cargo build` in the test output